src/SVGSkin.js

```diff
@@ -156,13 +156,17 @@ class SVGSkin extends Skin {
      * @param {Array<number>} [rotationCenter] - Optional rotation center for the SVG.
      */
     setSVG (svgData, rotationCenter) {
-        this._svgRenderer.loadSVG(svgData, false, () => {
-            const svgSize = this._svgRenderer.size;
-            if (svgSize[0] === 0 || svgSize[1] === 0) {
-                super.setEmptyImageData();
-                return;
-            }
+        this._svgRenderer.loadString(svgData, false /* fromVersion2 */);
+        const svgSize = this._svgRenderer.size;
+        if (svgSize[0] === 0 || svgSize[1] === 0) {
+            super.setEmptyImageData();
+            return;
+        }
+
+        if (typeof rotationCenter === 'undefined') rotationCenter = this.calculateRotationCenter();
```

Comment:
calculateRotationCenter () {
        return [this.size[0] / 2, this.size[1] / 2];
    }